src/main/java/com/easypost/model/Shipment.java

```diff
@@ -1090,21 +1102,29 @@ public static Smartrate getLowestSmartRate(final List<Smartrate> smartrates, int
      * @param deliveryAccuracy Delivery days accuracy restriction to use when filtering.
      * @return lowest Smartrate object
      * @throws EasyPostException when the request fails.
+     * @deprecated Use {@link #findLowestSmartrate(List, int, SmartrateAccuracy)} instead.
```

Comment:
Question: Shouldn't these have the names of the params? How does `@link` work?

Reply:
https://stackoverflow.com/questions/11458398/what-does-link-mean-in-java-android-documentation